modules/ROOT/pages/errors/gql-errors/42NAE.adoc

```diff
@@ -0,0 +1,32 @@
+= 42NAE
+
+== Status description
+error: syntax error or access rule violation - auth rule already exists. An auth rule with the name `{ <<authRule>> }` already exists.
+
+== Explanation
+When creating an authorization rule, the specified auth rule name must be unique.
+If an auth rule with the same name already exists, the operation will fail with this error.
+
+== Example scenario
+Try to create an auth rule using the following command:
+
+[source,cypher]
+----
+CREATE AUTH RULE authRuleName SET CONDTION abac.oidc.user_attribute('department') = 'sales';
+----
+If the auth rule `authRuleName` does already exist, an error will be thrown with GQLSTATUS 42NAD.
+
+== Possible solutions
+
+To resolve this issue, make sure that there is no existing auth rule with the same name before calling the create command.
+
+Another solution is to use the `IF NOT EXISTS` clause in the drop command to avoid the error if the auth rule does already exist. Then the auth rule will only be created if the name of the auth rule is unique.
```

Comment:
Do you mean create command instead of drop?
```suggestion
Another solution is to use the `IF NOT EXISTS` clause in the drop command to avoid the error if the auth rule already exists. 
Then the auth rule will only be created if the name of the auth rule is unique.
```